benchmarks/build.gradle.kts

```diff
@@ -62,10 +63,29 @@ android {
         )
     }
 
+    testOptions.managedDevices.devices {
+        create<com.android.build.api.dsl.ManagedVirtualDevice>("pixel6Api31") {
+            device = "Pixel 6"
+            apiLevel = 31
+            systemImageSource = "aosp"
+        }
+    }
+
```

Comment:
I think this device is already configured in `AndroidTestConventionPlugin` by using `configureGradleManagedDevices()`